base/int.jl

```diff
@@ -203,46 +198,28 @@ rem{T<:Integer}(x::T, ::Type{T}) = x
 rem(x::Integer, ::Type{Bool}) = ((x&1)!=0)
 mod{T<:Integer}(x::Integer, ::Type{T}) = rem(x, T)
 
-for to in (Int8, Int16, Int32, Int64)
-    @eval begin
-        convert(::Type{$to}, x::Float32) = box($to,checked_fptosi($to,unbox(Float32,x)))
-        convert(::Type{$to}, x::Float64) = box($to,checked_fptosi($to,unbox(Float64,x)))
+for Tf in (Float32, Float64)
+    for T in SignedIntTypes
+        @eval convert(::Type{$T}, x::$Tf) = box($T,checked_fptosi($T,unbox($Tf,x)))
     end
-end
-
-for to in (UInt8, UInt16, UInt32, UInt64)
-    @eval begin
-        convert(::Type{$to}, x::Float32) = box($to,checked_fptoui($to,unbox(Float32,x)))
-        convert(::Type{$to}, x::Float64) = box($to,checked_fptoui($to,unbox(Float64,x)))
+    for T in UnsignedIntTypes
```

Comment:
Yes, that was a typo, should have been Signed64Types and Unsigned64Types, instead of SignedIntTypes/UnsignedIntTypes.  Now fixed.